src/AddTodoForm.jsx

```diff
@@ -1,21 +1,36 @@
+import Button from "../public/components/Button";
 import PropTypes from "prop-types";
-Button.propTypes = {
-  children: PropTypes.node.isRequired,
+
+AddTodoForm.propTypes = {
+  onAddTodo: PropTypes.func.isRequired,
 };
 
-export default function AddTodoForm() {
+export default function AddTodoForm({ onAddTodo }) {
+  function handleAddTodo(e) {
+    e.preventDefault();
+
+    const newToDoList = e.target.elements.todoTitle.value.trim();
+    console.log(e.target.elements.todoTitle.value.trim());
+
+    const newToDoItem = {
+      id: Date.now(),
+      title: newToDoList,
+    };
+    console.log(newToDoItem);
+
+    onAddTodo(newToDoItem);
```

Comment:
It's expected to be the title alone, but let's keep it since it's a requirement from the next assignment (week 4).